pkgs/development/python-modules/expiringdict/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, python }:
+
+buildPythonPackage rec {
+  pname = "expiringdict";
+  version = "1.2.1";
+  disabled = !python.isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0l421pczpglfwqhl1kcb7wrfxg4554p6v7yjlg4c698448ks8azy";
+  };
+
+  doCheck = false;
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.